airtime_mvc/application/views/scripts/playlist/update.phtml

```diff
@@ -2,6 +2,35 @@
 $items = $this->contents;
 $isStaticSmartBlock = ($this->obj instanceof Application_Model_Block && $this->obj->isStatic());
 $isPlaylist = ($this->obj instanceof Application_Model_Playlist);
+if ($this->poolCount) { ?>
+<div class='sp_text_font sp_text_font_bold'>
```

Comment:
Some CSS might have gone missing, at least the text seems to have very low contrast. Most likely this got kicked during a redesign and never had it's style updated.

![image](https://user-images.githubusercontent.com/116588/34355055-e7250240-ea32-11e7-8f22-03cce54616a3.png)

We might want to consider cleaning this up for more contrast. I won't hold off on merging over this though.